spec/magma_spec.rb

```diff
@@ -1,10 +1,8 @@
 require_relative '../lib/magma'
 require 'yaml'
 
-def disconnect_attribute model, att_name
-  att = model.attributes[att_name]
-  model.attributes[att_name] = nil
-  att
+def disconnect_attribute(model, att_name)
+  model.attributes.delete(att_name)
```

Comment:
In production, it's pretty unlikely we'd define nil attributes on a model. So instead of leaving nil attributes behind in the tests, we can completely remove them.